Runner/Runner.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import print_function
+from msrestazure.azure_active_directory import AdalAuthentication 
+from azure.common.credentials import ServicePrincipalCredentials
+import azure.batch.batch_service_client as batch
+import azure.batch.batch_auth as batchauth
+import inspect
+import traceback
+import azure.batch
+import azure.batch.models.batch_error 
+import azure.common
+import logging
+import websockets
```

Comment:
💡 📓 Some of these are unused imports, I would encourage to use `flake 8` its very imple to run on your file and it will catch few things for you. Flake 8 link : http://flake8.pycqa.org/en/latest/

Reply:
Thanks I'll use flake 8. I was wondering what the best tool is for the job.